plume-models/src/blogs.rs

```diff
@@ -829,10 +829,13 @@ pub(crate) mod tests {
                 },
             );
 
+            println!("count bfore {}", blog[0].count_authors(conn));
             user[0].delete(conn, &searcher);
             assert!(Blog::get(conn, blog[0].id).is_some());
             assert!(Blog::get(conn, blog[1].id).is_none());
+            println!("count after bfore {}", blog[0].count_authors(conn));
             user[1].delete(conn, &searcher);
+            println!("count after after {}", blog[0].count_authors(conn));
```

Comment:
"bfore", "after bfore", "after after", then is it "after after bfore" or "after after after"?
(joking about the comments, but they should be removed actually, I guess you forgot to)

Reply:
Oops :grimacing: